main.py

```diff
@@ -1,10 +1,42 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player_name, player_data in data.items():
+        race = player_data["race"]
+        skills = race["skills"]
+        guild = player_data["guild"]
+
+        race = Race.objects.get_or_create(
+            name=race["name"],
+            description=race["description"]
+        )[0]
+
+        for skill in skills:
+            Skill.objects.get_or_create(
```

Comment:
you don't need `get_or_create` here so use simple `create`

Reply:
It won't work this way. 
A skill model's name must be unique - so if we parse the json file with just `createI()` we'll get an error because of the doubling skills from different players (we have 3 players with same skillsets).

Reply:
Instead implemented get_or_create() to player creation. 
(So the script won't fail in case we occasionally run the it several times or there is an existing user among new ones in a file)